samples/java/com/microsoft/bing/commerce/samples/TSManagementAPISample.java

```diff
@@ -0,0 +1,79 @@
+package com.microsoft.bing.commerce.samples;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import com.microsoft.bing.commerce.ingestion.BingCommerceIngestion;
+import com.microsoft.bing.commerce.ingestion.implementation.BingCommerceIngestionImpl;
+import com.microsoft.bing.commerce.ingestion.models.*;
+
+import com.microsoft.bing.commerce.search.BingCommerceSearch;
+import com.microsoft.bing.commerce.search.implementation.BingCommerceSearchImpl;
+import com.microsoft.bing.commerce.search.models.*;
+import com.microsoft.bing.commerce.search.util.AccessTokenInterceptor;
+import com.microsoft.bing.commerce.search.util.AccessTokenProvider;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+public class TSManagementAPISample {
+    public final static String TENANT_ID = System.getenv("TENANT_ID");
+    public final static String ACCESS_TOKEN = System.getenv("ACCESS_TOKEN");
+    public final static String INDEX_NAME = System.getenv("Sample_Index");
+
+    public static void main(String[] args) throws IOException {
+        BingCommerceIngestion ingestionClient = createIngestionClient();
+
+    }
+
+    private static void createScript(BingCommerceIngestion ingestionClient) throws IOException {
+        File file = new File("Defines a path to the content index on your local file system");
+        BufferedReader br = new BufferedReader(new FileReader(file));
+
+        String script = br.readLine();
+        br.close();
```

Comment:
This will only read one line from the file. Please choose an option from here and do it: https://www.journaldev.com/875/java-read-file-to-string